internal/file/proto_set_provider.go

```diff
@@ -164,8 +181,14 @@ func (c *protoSetProvider) getBaseProtoSets(dirPathToProtoFiles map[string][]*Pr
 		}
 		protoSet.DirPathToFiles[dirPath] = append(protoSet.DirPathToFiles[dirPath], protoFiles...)
 		var config settings.Config
-		// configFilePath is empty if no config file is found
-		if configFilePath != "" {
+		// we have already validated that configData is also set
+		if c.configDirPath != "" {
```

Comment:
We can update this to `c.configData != ""`

Reply:
Done